src/EnergyPlus/DaylightingManager.cc

```diff
@@ -9999,14 +9998,20 @@ void ReportIllumMap(EnergyPlusData &state, int const MapNum)
         SavedMnDy(MapNum) = state.dataEnvrn->CurMnDyHr.substr(0, 5);
     }
     if (EnvrnPrint(MapNum)) {
+        std::string refPt2;
+        if (state.dataDaylightingData->ZoneDaylight(state.dataDaylightingData->IllumMap(MapNum).Zone).TotalDaylRefPoints > 1) {
+            refPt2 = RefPts(state.dataDaylightingData->IllumMap(MapNum).Zone, 2);
```

Comment:
What's the magic 2 here?  Do we need a more dynamic daylighting map that can handle > 2 now as well?

Reply:
If I remember correctly, this is just for reference in the map header so you can see where your reference points would fall within the map grid.
```
Date/Time,DAYLIT ZONE:CHICAGO ANN CLG 1% CONDNS DB=>MWB:DAYLIT MAP at 0.80m Illuminance [lux] (Hourly),RefPt1=(2.50:5.00:0.80),RefPt2=(2.50:3.00:0.80),,
 07/21 05:00,(0.10;0.10)=,(0.63;0.10)=,(1.17;0.10)=,(1.70;0.10)=,(2.23;0.10)=,(2.77;0.10)=,(3.30;0.10)=,(3.83;0.10)=,(4.37;0.10)=,(4.90;0.10)=
(0.10;0.10)=,0,0,0,0,0,0,0,0,0,0
(0.10;1.19)=,0,0,0,0,0,0,0,0,0,0
```
So, we could leave it just listing the first two ref points, but it would be better to list all of them.

Reply:
@Myoldmopar @mjwitte Ok, I've committed another code mod that should address this so that all of the points are listed even if there are more than two.  Also cleaned up the transition rule document omission from a recent PR and something I missed in another documentation only PR.  But, hold on a little bit...I forgot that unit test was still running on my machine and I just got a clang-format issue.  Hoping the unit test comes back clean.  If it does, I'll fix the formatting issue and make another commit.  Hopefully this will be good to go then?

Reply:
@Myoldmopar @mjwitte Unit test came back clean.  Clang-format has now been run (cleaned up one minor issue) and CI is now running.